src/content/api/integration-requests.mdx

```diff
@@ -15,7 +15,7 @@ import CodePanel from '../../components/CodePanel.astro';
 
 An Integration Request allows Centrapay users to request the creation of an [Integration](/api/integrations/) on a Centrapay Account.
 
-## Model
+## Integration Model
```

Comment:
```suggestion
## Integration Request Model
```

I think it should be Integration Request because an Integration can be an Integration Request or a Managed Integration.